src/ca65/scanner.c

```diff
@@ -769,6 +769,101 @@ static void ReadIdent (void)
     }
 }
 
+static void ReadRawStringConst (void)
+/* Called if we found the chars: 'R"'
+** Now Read a raw literal string constant into SVal.
+*/
+{
+    /* Space to store the keyword delimiter */
+    StrBuf* D = NewStrBuf ();
+
+    /* Skip the leading " string terminator */
+    NextChar ();
+
+    /* See if there is a delimiter defined after the: "
+    ** The delimiter string will follow the same rules as an identifier, but leading chars can be numeric.
+    */
+    while (IsIdChar (C)) {
+        /* Add character to the delimiter */
+        SB_AppendChar (D, C);
+        NextChar ();
+    }
+
+    /* next char after the delimiter has to be '(' */
+    if (C != '(') {
+        Error ("Error in raw string. Expected: `('");
+    }
+
+    /* skip the '(' */
+    NextChar ();
+
+    /* Read the string */
+    while (1) {
+
+        /* Maybe we are done ? */
+        if (C == ')') {
+            NextChar ();
+
+            /* If we had a opening delimiter, try a match to previous delimiter */
+            if (D->Len)
+            {
+                /* Does the first char match the first char of the stored delimiter ? */
+                if (C == D->Buf[0]) {
```

Comment:
Okay that makes sense, I'll go that route. If I work on this today it will be late this evening (EST).